docs/getting-started/cluster-setup.md

```diff
@@ -0,0 +1,235 @@
+A cluster consists of a master API server and a set of worker VMs called nodes.
+
+#### Using get-kube
+To start up your cluster, run:
+
+```bash
+#Using wget
+export KUBERNETES_PROVIDER=aws; wget -q -O - https://get.k8s.io | bash
+#Using cURL
+export KUBERNETES_PROVIDER=aws; curl -sS https://get.k8s.io | bash
+```
+
+This process takes about 5 to 10 minutes. User credentials and security tokens are written in `~/.kube/config`.
+By default, the script will provision a new VPC and a 4 node k8s cluster in `us-west-2a` (Oregon) with EC2 instances running on Debian.
+
+NOTE: You can override the variables defined in `config-default.sh` to change this behavior. See the [Getting Started with Kubernetes guide](http://kubernetes.io/docs/getting-started-guides/aws/) for more details.
+
+
+
+### Setup secrets
+A Secret is an object that contains a small amount of sensitive data such as a password, a token, or a key. Such information can be put in a Pod specification or in an image, but putting it in a Secret object allows for more control over how it is used, and reduces the risk of accidental exposure. You can create a secret object in a file first, in json or yaml format, and then create that object.
+
+First, you will need to acquire your secrets.
+
+#### Kubernetes secrets
+Kubectl can be used to see your [Kubernetes secrets](http://kubernetes.io/docs/user-guide/secrets/walkthrough/).
+To get your k8s `host`, run:
+```bash
+$ kubectl cluster-info
+
+Kubernetes master is running at https://55.555.55.55
+```
+
+To get the k8s `<DEFAULT_TOKEN_NAME>`, run:
+```bash
+$ kubectl get secrets
+
+NAME                      TYPE                                  DATA      AGE
+datastore-dynamodb        Opaque                                2         19d
+db-user-pass              Opaque                                2         49d
+default-token-abc55       kubernetes.io/service-account-token   3         50d
+git-token                 Opaque                                1         42d
+mysecret                  Opaque                                2         49d
+npm-token                 Opaque                                1         42d
+plugin-login-secrets      Opaque                                4         44d
+screwdriver-job-secrets   Opaque                                1         43d
+```
+
+The `<DEFAULT_TOKEN_NAME>` will be listed under `Name` when the `Type` is `kubernetes.io/service-account-token`.
+
+To get your k8s `token`, run:
+```bash
+$ kubectl describe secrets <DEFAULT_TOKEN_NAME>  # default-token-abc55 in this example
+```
+
+#### DynamoDB secrets
+To get your `accessKeyId` and `secretAccessKey`, navigate to [IAM](https://console.aws.amazon.com/iam)
+in your AWS console. Click on Users, and create a Screwdriver user with `<USERNAME>`. We recommend installing using an account which has full access to the AWS APIs. Select the `Security Credentials` tab and click `Create Access Key`. Download the file and keep note of those values.
```

Comment:
I think this list would be better presented as a numbered list of steps in order to get the dynamo secrets